src/components/dialogs/create-study-dialog/create-study-dialog.js

```diff
@@ -60,58 +63,50 @@ const CreateStudyDialog = ({ open, onClose, providedExistingCase }) => {
     const selectedDirectory = useSelector((state) => state.selectedDirectory);
     const userId = useSelector((state) => state.user.profile.sub);
 
-    const schema = yup.object().shape({
-        [STUDY_NAME]: yup.string().required(),
-        [FORMATTED_CASE_PARAMETERS]: yup.mixed(),
-        [DESCRIPTION]: yup.string().nullable(),
-        [CURRENT_PARAMETERS]: yup.mixed(),
-        [CASE_UUID]: yup.string(),
-        [CASE_FILE]: yup.mixed(),
-    });
-
     const createStudyFormMethods = useForm({
         mode: 'onChange',
-        defaultValues: getCreateStudyDialogFormDefaultValues(),
-        resolver: yupResolver(schema),
+        defaultValues: getCreateStudyDialogFormDefaultValues({
+            activeDirectory,
+        }),
+        resolver: yupResolver(createStudyDialogFormValidationSchema),
     });
 
     const {
         setValue,
         formState: { errors },
         setError,
-        watch,
         clearErrors,
+        getValues,
+        watch,
     } = createStudyFormMethods;
 
     // Constants
-    const caseFileErrorMessage = errors.caseFile?.message;
-    const apiCallErrorMessage = errors.apiCall?.message;
+    const apiCallErrorMessage = errors.root?.apiCall?.message;
     const studyNameErrorMessage = errors.studyName?.message;
 
-    const caseFile = watch(CASE_FILE);
-    const caseUuid = watch(CASE_UUID);
-    const formattedCaseParameters = watch(FORMATTED_CASE_PARAMETERS);
+    const { caseFile, caseUuid, formattedCaseParameters } = getValues();
```

Comment:
Same as above, delete this getValues.
I see that `caseFile` is only used to pass it to `UploadNewCase`... so why do you get it here ? Just remove it no ?
`caseUuid` is only used in `handleDeleteCase`. So you should use the getValues from here. And then no need to use `watch` for it anymore.